### PR TITLE
First shot at 'Library Organization'

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

**Comment by WarrenWeckesser** [comment]
Consider this a very rough draft.  Is the content what you had in mind for this section?  How much detail do we want here?

**Comment by jarrodmillman** [comment]
This seems like the right level of detail to me.  I would add a `\cite{peterson2009f2py}` somewhere and remove the fixme.  Any other comments/suggestions @stefanv?

File: paper.tex [hunk 1]
```diff
@@ -134,6 +134,95 @@ \section*{History}
 \section*{Architecture and implementation choices}
 
 \subsection*{Library organisation}
+Broadly, the NumPy library consists of the following parts:
+the NumPy array data structure; the so-called \emph{universal functions};
```
Comment (stefanv) [inline paper.tex]:
```suggestion
the NumPy array data structure; the so-called \emph{universal functions} (described below);
```

**Comment by stefanv** [comment]
I have some minor changes, but I'll just make those in a subsequent PR; I agree that this is about the right granularity.

**Pull request merged**